source/agora/common/Version.d

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+
+    Contains versioning info
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.common.Version;
```

Comment:
Why this module ?

Reply:
Useless actually, can remove it for now.